pico_limbo/src/configuration/boss_bar.rs

```diff
@@ -0,0 +1,89 @@
+use minecraft_packets::play::boss_bar_packet::{BossBarColor, BossBarDivision};
```

Comment:
No data from the packets crate should be used here
If needed, do a conversion from the config type to another type in the server state builder